### ISSUE TITLE
Bug in example? Or precision on ttg_fence()

### ISSUE THREAD

**Issue by therault** [TESSEorg/ttg#88]
In examples/floyd-warshall/floyd_warshall.cc (and a few others), we do
```
606       ttg::ttg_initialize(argc, argv);
607       ttg_fence(ttg_default_execution_context());
```
The PaRSEC driver complains:
```
parsec_context_wait detected on a non started context
```
This is because fence() is implemented with completing the TTG parsec taskpool, which has not been started, since nothing has been scheduled in this taskpool yet.

In the current parsec implementation, the legal sequence would be
```
   ttg::ttg_initialize(argc, argv);
   // Create Ops, start() some of them
   ttg_fence(ttg_default_execution_context());
```

1- Do we want to authorize calls to `ttg_fence()` at any time?
2- Should this have the semantics of a synchronizing fence all the time (i.e. equivalent to an `MPI_Barrier()`)?

**Comment by devreal** [comment]
@therault Was this fixed in #95?

**Comment by therault** [comment]
Yes it was. Closing.
